schema205/json_translate.py

```diff
@@ -96,16 +106,58 @@ def add_data_group(self, group_name, group_subdict):
             if 'Notes' in element:
                 elements['properties'][e]['notes'] = element['Notes']
             if 'Required' in element:
-                required.append(e)
+                #required.append(e)
+                req = element['Required']
+                if isinstance(req, bool) and req == True:
+                    required.append(e)
+                elif req.startswith('if'):
+                    if '!=' in req:
+                        self._construct_requirement_if_else(elements, req.split(' ')[1].split('!')[0],
+                                                            False, req.split('=')[1], e)
+                    elif '=' in req:
+                        self._construct_requirement_if_else(elements, req.split(' ')[1].split('!')[0],
```

Comment:
This line isn't quite working correctly. In RS0003, it produces:

```json
"if": {
    "properties": {
        "is_enclosed=True": {
            "const": "True"
        }
    }
},
"then": {
    "required": [
        "assembly_components"
    ]
},
```

When (I think?) it should be:

```json
"if": {
    "properties": {
        "is_enclosed": {
            "const": true
        }
    }
},
"then": {
    "required": [
        "assembly_components"
    ]
},
```